client/webpack.config.js

```diff
@@ -376,13 +376,14 @@ const webpackConfig = {
 			: [] ),
 
 		/*
-		 * When not available, replace languages-meta.json with fallback-languages-meta.json.
+		 * When available, replace fallback-languages-meta.json with languages-meta.json
 		 */
 		hasLanguagesMeta &&
 			new webpack.NormalModuleReplacementPlugin(
-				/^languages[/\\]fallback-languages-meta.json$/,
-				'languages/languages-meta.json'
+				new RegExp( path.join( __dirname, 'languages/fallback-languages-meta.json' ) ),
```

Comment:
Nitpick:

```suggestion
				new RegExp( path.join( __dirname, 'languages', 'fallback-languages-meta.json' ) ),
```

Reply:
I did some tests and `path.join(a, "b/c")` will still produce a valid path in windows (`a\\b\\c`). Was that your concern?

Reply:
Yup, feel free to disregard, then 👍